packages/backend/src/matter/devices/climate-device.ts

```diff
@@ -55,6 +60,7 @@ const ClimateDeviceType = (
     BasicInformationServer,
     IdentifyServer,
     HomeAssistantEntityBehavior,
+    OnOffServer.set({ config: climateOnOffConfig }),
```

Comment:
This should only be exposed if the entity's `supported_features` includes `TURN_ON` and `TURN_OFF`.

https://developers.home-assistant.io/docs/core/entity/climate/#supported-features